### ISSUE TITLE
Install issue

### ISSUE THREAD

**Issue by vouill** [simpulton/ngrx-rest-app#11]
Hey,

I have some dependency troubles while installing for the first time.

```
npm ERR! Linux 4.2.0-35-generic                                                                  
npm ERR! argv "/usr/bin/nodejs" "/usr/bin/npm" "i"                                               
npm ERR! node v4.4.3                                                                             
npm ERR! npm  v2.15.1                                                                            
npm ERR! code EPEERINVALID                                                                       
                                                                                                
npm ERR! peerinvalid The package rxjs@5.0.0-beta.7 does not satisfy its siblings' peerDependencie│
s requirements!                                                                                  
npm ERR! peerinvalid Peer @angular/core@2.0.0-rc.1 wants rxjs@5.0.0-beta.6                       
npm ERR! peerinvalid Peer @angular/http@2.0.0-rc.1 wants rxjs@5.0.0-beta.6                       
npm ERR! peerinvalid Peer @ngrx/store@1.5.0 wants rxjs@5.0.0-beta.6  
```

npm-debug : 
```
157971 error Linux 4.2.0-35-generic
157972 error argv "/usr/bin/nodejs" "/usr/bin/npm" "i"
157973 error node v4.4.3
157974 error npm  v2.15.1
157975 error code EPEERINVALID
157976 error peerinvalid The package rxjs@5.0.0-beta.7 does not satisfy its siblings' peerDependencies requirements!
157976 error peerinvalid Peer @angular/core@2.0.0-rc.1 wants rxjs@5.0.0-beta.6
157976 error peerinvalid Peer @angular/http@2.0.0-rc.1 wants rxjs@5.0.0-beta.6
157976 error peerinvalid Peer @ngrx/store@1.5.0 wants rxjs@5.0.0-beta.6
157977 verbose exit [ 1, true ]
```
Anyone encountered the same issue ?

**Comment by jdgarvey** [comment]
@vouill you appear to have Node version < 5.x and NPM < 3.x. Please update to at least Node 5.x and NPM 3.x and try installing modules again. Thanks!

**Comment by vouill** [comment]
@jdgarvey  Hey !

I did upgraded node and npm and it did the trick, thanks!

However i faced another issue.
`typings install` would give me no results, meaning nothing got installed by typings and the bundling then had some issues too.
It comes from the new  [typings](https://github.com/typings/typings) changes.

I only had to change : 
tsconfig.json: 
```
-"typings/browser.d.ts",
+"typings/index.d.ts",
```

typings.json : 
```
-"ambientDependencies": {
+ "globalDependencies": {
```

Hope it helps !
